lib/unexpectedMitm.js

```diff
@@ -859,7 +940,7 @@ module.exports = {
                 }).spread(function (fulfilmentValue, httpConversation, httpConversationSatisfySpec) {
                     expect.errorMode = 'default';
                     return expect(httpConversation, 'to satisfy', httpConversationSatisfySpec).then(function () {
-                        if (expect.flags['with extra info']) {
+                        if (expect.flags['and verified'] || expect.flags['with extra info']) {
                             return [fulfilmentValue, httpConversation, httpConversationSatisfySpec];
```

Comment:
What I've done is make sure it fulfils with the same format as 'with extra info' since that's what it consumes. The change to make fulfilmentValue from the delegated assertion the first arg went in, so the API should at least be consistent.